### PR TITLE
IDeal and Sofort

### PR DESCRIPTION
**Pull request by cyberxander90**
Support **OnlineBanking** payment methods with **iDeal** and **sofort**.

### CODE REVIEW

File: lib/recurly/adyen-token.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  token (type, payload) {
+    debug('Invoking AdyenToken tokenization');
+
+    const errors = validatePayload(payload);
+    if (errors.length > 0) {
+      return this.error('validation', { fields: errors });
+    }
+
+    this.recurly.request.post({
+      route: '/adyen_token/token',
```
Comment (brittany-wood) [inline lib/recurly/adyen-token.js:44]: this needs to be changed to `adyen/token`

File: lib/recurly/adyen-token.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    return err;
+  }
+
+  loadIdealBanks ({
+    data: { currency, amount, countryCode } = {},
+    done,
+    attachTo,
+  } = {}) {
+    this.recurly.request.get({
+      route: '/adyen_token/ideal_banks',
```
Comment (brittany-wood) [inline lib/recurly/adyen-token.js:69]: this needs to be changed to `adyen/list_banks`

File: lib/recurly/adyen-token.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      data: {
```
Comment (brittany-wood) [inline lib/recurly/adyen-token.js:70]: only parameter needed here should be `payment_method_type` and we need to pass in `ideal`

**Review by brittany-wood — APPROVED** [review]

**Review by efeygelson — APPROVED** [review]

**Review by chrissrogers — APPROVED** [review]

**Review by horozcorecurly — APPROVED** [review]

**Comment by brittany-wood** [comment]
QA +1

**Pull request merged**